rust/operator-binary/src/kafka_controller.rs

```diff
@@ -960,6 +960,13 @@ fn build_broker_rolegroup_statefulset(
         ..EnvVar::default()
     });
 
+    // Needed for the `containerdebug` process to log it's tracing information to.
+    env.push(EnvVar {
+        name: "CONTAINERDEBUG_LOG_DIRECTORY".to_string(),
+        value: Some(format!("{STACKABLE_LOG_DIR}/containerdebug")),
+        value_from: None,
+    });
```

Comment:
Don't love the level of action-at-a-distance going on here.. :/ But I guess it's kind of unavoidable without doing a bigger restructuring.

Reply:
Yes. I also dislike that it's so decoupled from the command but I wanted to make it consistent and (somewhat more) transparent.